src/credentials/FabricTable.h

```diff
@@ -820,8 +834,24 @@ class DLL_EXPORT FabricTable
         return err;
     }
 
+    // For test only. See definition of `StateFlags::kAbortCommitForTest`.
+    void SetForceAbortCommitForTest(bool abortCommitForTest)
+    {
+        (void) abortCommitForTest;
+#if CONFIG_BUILD_FOR_HOST_UNIT_TEST
+        if (abortCommitForTest)
+        {
+            mStateFlags.Set(StateFlags::kAbortCommitForTest);
+        }
+        else
+        {
+            mStateFlags.Clear(StateFlags::kAbortCommitForTest);
+        }
+#endif // CONFIG_BUILD_FOR_HOST_UNIT_TEST
+    }
+
 private:
-    enum class StateFlags : uint8_t
+    enum class StateFlags : uint16_t
```

Comment:
We don't need this size increase until we add our next flag, right?

Reply:
Just trying to keep ABI compatibility for as long as I can